src/rpc/methods/f3/types.rs

```diff
@@ -834,4 +933,19 @@ mod tests {
         let serialized = serde_json::to_value(cert.clone()).unwrap();
         assert_eq!(lotus_json, serialized);
     }
+
+    #[test]
+    fn test_f3_manifest_parse_contract_return() {
+        // The solidity contract: https://github.com/filecoin-project/f3-activation-contract/blob/master/contracts/F3Parameters.sol
```

Comment:
```suggestion
        // The solidity contract: https://github.com/filecoin-project/f3-activation-contract/blob/063cd51a46f61b717375fe5675a6ddc73f4d8626/contracts/F3Parameters.sol
```
In case some re-org would happen in that repo.